server/models/comment.js

```diff
@@ -15,7 +15,7 @@ const CommentSchema = new Schema({
   },
   commenter: {
     type: Schema.Types.ObjectID,
-    ref: "user",
+    ref: "profile",
     required: true
   },
```

Comment:
This is clever but it may break things.

Reply:
I've made sure to change all the refs to profile, to avoid repeating the work of getting a user than finding it's profile. I've tested my changes and I think it works fine for now. If you notice anything broken, feel free to let me know and I will fix it.